File: cpp_e9zMZ/cpp/06-thaphinh/19.cpp
#include <iostream>

#include "zvn.h"
using namespace std;
int main() {
    int height = 7;
    string character = "#";
    string space = " ";
    string shape = "";
    string line = myRepeat(character, height);
    for (int i = 2; i < height; i++) {
        string record = "";
        for (int j = 1; j <= height; j++) {
            if (j == i || j == height - i + 1) {
                record += character;
            } else {
                record += space;
            }
        }
        shape += record + "\n";
    }
    shape = line + "\n" + shape + line;
    cout << shape << endl;
}

File: python_ToSXs/python/06-thaphinh/19.py
height = 5
shape = ""
shapeTop = ""
shapeBottom = ""
character = "#"
center = int(height / 2)

line = character * height
for i in range(2, center + 1):
    record = (" " * (i - 1) + character +
              " " * (height - 2 * i) +
              character)
    shapeTop += record + "\n"

centerCharacter = " " * center + character + "\n"
for i in range(2 + center, height):
    record = (" " * (height - i) +
              character +
              " " * (2 * i - height - 2) +
              character)
    shapeBottom += record + "\n"

shape = line + "\n" + shapeTop + centerCharacter + shapeBottom + line
print(shape)
